sheet4DiegoXavi.tex

```diff
@@ -0,0 +1,145 @@
+We are working on a way to solve that problem or show that indeed you are forced to use such convex hull algorithm. 
+
+Recall that the lower faces of $Q^c$ are those whose valid inequality is given by a hyperplane $h$ with $h_{d+1} < 0$ and a vector $h_0$, with $x \cdot h \leq h_0$ for all $x \in Q^c$.
+
+We do the following: 
+
+\begin{enumerate}
+	\item For each vertex $x_i \in P$, compute $\pi^c(x_i)$, so then it is $Q^c = conv\{\pi^c(x_i)\}$.
+	
+	\item Use a convex hull algorithm to find the actual vertices of $Q^c$.
+	
+	\item For all subsets of $d+1$ vertices:
+	\begin{enumerate}
+		\item Check if they are in the same hyperplane.
+		if they are, compute the hyperplane $(h,h_0)$ with $h_{d+1} < 0$
+		
+		\item For all other points $v$, check if $v \cdot h \leq h_0$:
+		\begin{enumerate}
+			\item If not, then it cannot be a lower facet, so we stop.
+			\item If $v \cdot h = h_0$, then $v$ is in the hyperplane, so we add $v$ to the subset, and compute the convex hull of this new subset to find the real facet.
+		\end{enumerate}
+		If at the end, all $v$ verify $v \cdot h \leq h_0$, then we have a valid inequality, and thus a face, and this face will be a lower face by definition.
+				
+	\end{enumerate}
+		
+\end{enumerate}
+
+
+
+\subsubsection*{Solution 2}
+Since $f$ is linear, we can write $f(x)=Ax$ for some matrix $A$. Then,
+\[ f(x) = Ax = \left( \sum_{i=1}^{n} a_{1i}x_i, \dots, \sum_{i=1}^{n} a_{ni}x_i \right) = (f_1(x), \dots, f_n(x)). \]
+For each coordinate $j=1,\dots,n$, using the linearity of the integral, 
+\[ \int_{Q}^{} f_j(x) \ dx = \int_{Q} \sum_{i=1}^{n} a_{ji}x_i \ dx = \sum_{i=1}^{n} a_{ji} \int_{Q} x_i dx = \sum_{i=1}^{n} a_{ji} \cdot r_{0i} = \text{vol}(Q)  A_j \cdot r_0, \]
+where $A_j$  is the $j$th row of $A$. Then, 
+\[ \int_{Q}^{} f(x) \ dx = (A_1\cdot r_0 \text{vol}(Q), \dots, A_n\cdot r_0 \text{vol}(Q)) = A \cdot r_0 \cdot \text{vol}(Q) = f(r_0) \cdot \text{vol}(Q). \]
+\subsubsection*{Solution 3}
+
+We follow the proof formally proving Ziegler's claims. 
+
+First of all, any convex combination of two sections is a section again.
+If $\gamma_1, \gamma_2$ are both sections of $\pi$, then for any $t\in [0,1]$, since $\pi$ is linear,
+$$\pi \circ (t\gamma_1+(1-t)\gamma_2)= t(\pi\circ \gamma_1) + (1-t)(\pi \circ \gamma_2) = t\cdot \text{id}+(1-t)\cdot\text{id}=\text{id}.$$
+
+Now, by the linearity of the integral, $\Sigma(P,Q)$ is convex. 
+If $p,q\in \Sigma(P,Q)$, then we can write
+$$
+\begin{array}{cc}
+ p=\frac{1}{\text{vol}(Q)}\int_Q \gamma_p(x) dx, & q=\frac{1}{\text{vol}(Q)}\int_Q \gamma_q(x) dx
+\end{array}
+$$
+and for any $t\in[0,1]$, 
+$$tp+(1-t)q = \frac{1}{\text{vol}(Q)}\int_Q (t\gamma_p + (1-t)\gamma_q)(x) dx \in \Sigma(P,Q)$$
+Now, the dimension of $\Sigma(P,Q)$ cannot be larger than $\dim P - \dim Q$, since $\Sigma(P,Q)\subset \pi^{-1}(r_0)$, which has this dimension. Here there are two assertions not proven. 
```

Comment:
"there are two assertions not proven" isn't English...
"there are two unproven assertions" is more like it!